packages/backend/src/payment-method/handler/service.ts

```diff
@@ -9,6 +9,7 @@ import { Transaction } from 'objection'
 
 export interface StartQuoteOptions {
   quoteId?: string
+  tenantId?: string
```

Comment:
The payment method handler for ILP doesn't seem to use this when it's getting a quote from the ILP network. Is this a necessary addition to the `StartQuoteOptions` interface?